src/target/arc_ocd.c

```diff
@@ -141,6 +141,13 @@ int arc_ocd_deassert_reset(struct target *target)
 	/* deassert reset lines */
 	jtag_add_reset(0, 0);
 
+	if (!target->reset_halt && target->state == TARGET_RESET)
+	{
```

Comment:
Rest of the file puts opening curly brace for conditions and loops at the same line as statement, only function definitions put opening curly brace on the next line.